pkg/northbound/topo/service.go

```diff
@@ -113,7 +114,59 @@ func (s *Server) Read(ctx context.Context, request *topoapi.ReadRequest) (*topoa
 	}, nil
 }
 
-// StreamChannel :
-func (s *Server) StreamChannel(topo.Topo_StreamChannelServer) error {
+// Subscribe ...
+func (s *Server) Subscribe(stream topo.Topo_SubscribeServer) error {
+	waitc := make(chan struct{})
+	go func() {
+		for {
+			subscribeRequest, err := stream.Recv()
+			if err == io.EOF {
+				//return nil
+				close(waitc)
+				return
+			}
+			if err != nil {
+				//return err
+				close(waitc)
+				return
+			}
+			log.Infof("Subscribe request %v", subscribeRequest)
+			// TODO - define subscribe request api
+		}
+	}()
+
+	ch := make(chan *Event)
+	if err := s.objectStore.Watch(ch); err != nil {
```

Comment:
Watch would need to be updated with a "withoutReplay" option - this is done for other stores in onos-config, as atomix can easily handle it